modules/sdk-core/src/bitgo/keychain/iKeychains.ts

```diff
@@ -85,6 +87,8 @@ export interface AddKeychainOptions {
   commonPub?: string;
   commonKeychain?: string;
   encryptedPrv?: string;
+  // Required for MPCV2 keys where we reduce the amount of data needed for the keycard
+  reducedEncryptedPrv?: string;
```

Comment:
Do we really need this in the input for this?

Reply:
nope we do not, removed